CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
         - order unsent reports by confirmed date
     - Bugfixes
 	- Application user in Docker container can't install packages. #2914
+    - Added junction lookup, so you can search for things like "M60, Junction 2"
```

Comment:
Looking at previous releases, I guess this should be under a "UK:" bullet point.